extensions/smallrye-graphql/deployment/src/main/java/io/quarkus/smallrye/graphql/deployment/SmallRyeGraphQLProcessor.java

```diff
@@ -129,6 +131,15 @@ void registerNativeImageResources(BuildProducer<ServiceProviderBuildItem> servic
                 lookupService);
         serviceProvider.produce(
                 new ServiceProviderBuildItem(LookupService.class.getName(), lookupImplementations.toArray(new String[0])));
+
+        // Schema Extension Service (We use the one from the CDI Module)
+        String schemaExtensionService = SPI_PATH + SchemaBuildingExtensionService.class.getName();
+        Set<String> schemaExtensionImplementations = ServiceUtil.classNamesNamedIn(
+                Thread.currentThread().getContextClassLoader(),
+                schemaExtensionService);
+        serviceProvider.produce(
+                new ServiceProviderBuildItem(SchemaBuildingExtensionService.class.getName(),
+                        schemaExtensionImplementations.toArray(new String[0])));
```

Comment:
Would it be hard to add a test for this feature?

Reply:
Good idea. Let me look into this.

Reply:
Done